tasks/github_tasks.py

```diff
@@ -270,3 +279,34 @@ def assign_team_label(_, pr_id=-1):
                 print(label_name, 'label assigned to the pull request')
             except github.GithubException.GithubException:
                 print(f'Failed to assign label {label_name}')
+
+
+@task
+def notify_community_pr(_, repo='', pr_id=-1):
```

Comment:
**❓ question:** Why do we need the default values for the parameters?

Reply:
It is used to auto cast parameters to the target type with invoke (otherwise `pr_id` would be a `str`)